ai-automations/styles/globals.css

```diff
@@ -94,45 +90,54 @@ body {
   --markprompt-ring: #fff;
 }
 
-:where([class^='Markprompt']) {
+/* 
+We don't use @markprompt/css directly because :where() clauses get overridden by some of Tailwind's defaults.
+Instead, we copy over @markprompt/css' styles and remove the where clauses.
+*/
```

Comment:
@michaelfester this file shouldn't be removed, rationale here

Reply:
Oh I missed this comment. Would there be any reasonable way to address this? Would be nice if Tailwind projects don't need to copy the entire CSS.

Reply:
Hmmm, I was considering seeing if we could create a plugin for PostCSS that just removes the `:where()` statements. That said, I also think our CSS package is due a refactor. It's getting fairly large and it can be cleaned up a lot, might also be a good time to  see if there's something besides `:where` selectors that would also work with Tailwind.